packages/react-components/src/components/ComplianceBanner/Utils.test.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { computeVariant } from './Utils';
+
+describe('Record and Transcription tests', () => {
+  describe('computeVariant tests', () => {
+    test('start recording and transcription state remains off, computeVariant should return RECORDING_STARTED', () => {
+      expect(computeVariant('on', 'off')).toEqual('RECORDING_STARTED');
+    });
+    test('start transcribing but recording state remains off, computeVariant should return TRANSCRIPTION_STARTED', () => {
+      expect(computeVariant('off', 'on')).toEqual('TRANSCRIPTION_STARTED');
+    });
+    test('stop recording and transcribtion state remains off, computeVariant should return RECORDING_STOPPED', () => {
```

Comment:
small spelling mistake here:
transcribtion -> transcription

Reply:
Fixed in a follow up PR.